docs/source/checks/tabular/model_evaluation/plot_confusion_matrix_report.py

```diff
@@ -4,6 +4,20 @@
 
 Confusion Matrix Report
 ***********************
+This notebook provides an overview for using and understanding the Confusion Matrix Report check.
+
+
+**Structure:**
+
+* `What is the Confusion Matrix Report? <#what-is-the-confusion-matrix-report>`__
+* `Generate data & model <#generate-the-data>`__
+* `Run the check <#run-the-check>`__
+
+
+What is the Confusion Matrix Report?
+====================================
+The ``ConfusionMatrixReport`` produce a confusion matrix visualization which help to assess
+the performance of the model (`confusion matrix <https://en.wikipedia.org/wiki/Confusion_matrix>`_).
```

Comment:
Maybe add another sentence on what a confusion matrix is?

Reply:
looks good?

`The ``ConfusionMatrixReport`` produces a confusion matrix visualization which summarize
the performance of the model. The confusion matrix contains the TP (true positive), FP (false positive), 
TN (true negative) and FN (false negative), from which we can derive the relevant metrics, such as accuracy, 
precision, recall etc. (`confusion matrix <https://en.wikipedia.org/wiki/Confusion_matrix>`_).`

Reply:
Yeah, looks good!
*summarizes